### ISSUE TITLE
Season States Already Downloaded

### ISSUE THREAD

**Issue by flip1900** [p-hueber/prefetcharr#63]
Hello, wondering if this is a simple issue i can use some help to sort of.
When prefetcharr detects im finishing watching a season , it says the next season is already downloaded when it isnt, what can be wrong in my configuration for this to happen? I cannot have the next season monitored on sonarr despite not being downloaded ??

**Comment by p-hueber** [comment]
What's Sonarr telling you about the next season? Is it listed and does it contain any episodes?
Prefetcharr just checks `episodes_count == total_episode_count` to determine whether a season is downloaded or not. I never observed this case, but if both are zero it wouldn't do what we want.
Do you mind sharing show and season in question?

**Comment by flip1900** [comment]
Prefetcharr is stating already downloaded, but it wasnt.
The show in question was Billions and Season 2 and 3 happened this

Cumprimentos,
Luís Correia
________________________________

**Comment by p-hueber** [comment]
I might be using the wrong episode counts here. I assume you had all episodes of the next season monitored already?
Please try unmonitoring all future seasons to work around this issue for now.

**Comment by flip1900** [comment]
Yes all seasons were monitored. I added the show with all monitored.
I dont have an issue with manually downloading this show as im the one watching it.
My usecase with prefetcharr is to when friends and family request a show i only download the first season and then let prefetcharr download the rest as they keep on watching. Let me know if there is something i can do to help you with this issue.

Cumprimentos,
Luís Correia
________________________________

**Comment by p-hueber** [comment]
This is a pretty straight forward fix, I'll leave a comment here when it's released.
Thanks for reporting!

**Comment by flip1900** [comment]
Of course, as soon as it updates and im near the end of watching a season i will let you know if it works

Cumprimentos,
Luís Correia

________________________________
De: Paul Hüber ***@***.***>
Enviado: domingo, dezembro 1, 2024 9:30:34 PM
Para: p-hueber/prefetcharr ***@***.***>
Cc: flip1900 ***@***.***>; Mention ***@***.***>
Assunto: Re: [p-hueber/prefetcharr] Season States Already Downloaded (Issue #63)


This should be addressed in 0.8.2.
@flip1900<https://github.com/flip1900> I'd love to get some feedback on whether or not this fixes your issue.

**Comment by flip1900** [comment]
Hello, just to update you, it works properly now.
Nice holidays and thanks for the help!

Cumprimentos,
Luís Correia
________________________________
